satellite/rewards/partners.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package rewards
+
+// PartnerInfo contains the name and ID of an Open Source Partner
+type PartnerInfo struct {
+	ID, Name string
+}
+
+// FormattedName returns formatted partner name
+func (p *PartnerInfo) FormattedName() string {
+	return p.ID + "-" + p.Name
+}
+
+// Partners contains a list of partners.
+type Partners map[string]PartnerInfo
+
+// LoadPartnerInfos returns our current Open Source Partners.
+func LoadPartnerInfos() Partners {
```

Comment:
Maybe a comment here to say that this is a mock and once we have the actual partner id stored on the satellite, we will implement the actual function call?